app/3.0/docs/upgrade.md

```diff
@@ -1546,7 +1546,7 @@ separate, optional imports. These include:
 -   [`<dom-bind>` element](devguide/templates#dom-bind)
 -   [`<dom-if>` element](devguide/templates#dom-if)
 -   [`<dom-repeat>` element](devguide/templates#dom-repeat)
--   [`Polymer.RenderStatus`](/{{{polymer_version_dir}}}/docs/api/namespaces/Polymer.RenderStatus)
+-   [`Polymer.RenderStatus`](/{{{polymer_version_dir}}}/docs/api/utils/render-status)
```

Comment:
naming of utils?

Reply:
Oh, just realized this is the upgrade guide, which needs to be totally wiped. Will leave this one alone.